.pre-commit-config.yaml

```diff
@@ -64,27 +64,26 @@ repos:
     args: ['-c', '"Google LLC"', '-l', 'apache']
     exclude: docs/videos/healthcare-and-life-sciences/lysozyme-example/submit.sh
     pass_filenames: true
+  - id: gocyclo
+    name: gocyclo
+    language: system
+    entry: gocyclo
+    args: ['-over', '15']
+    files: '\.go$'
+    pass_filenames: true
 
-- repo: https://github.com/dnephin/pre-commit-golang
-  rev: v0.5.1
-  hooks:
-  - id: go-fmt
-  - id: go-vet
-  - id: go-imports
-  - id: go-cyclo
-    args: [-over=15]
-  - id: go-unit-tests
-  - id: go-build
-  - id: go-mod-tidy
 - repo: https://github.com/tekwizely/pre-commit-golang
   rev: v1.0.0-rc.1
   hooks:
   - id: go-critic
-    args: [-disable, "#experimental,sloppyTypeAssert"]
+  - id: go-mod-tidy
 - repo: https://github.com/Bahjat/pre-commit-golang
   rev: v1.0.2
   hooks:
+  - id: go-fmt-import
```

Comment:
Could the Bahjat hooks be replaced with:

- https://github.com/TekWizely/pre-commit-golang#go-fmt
- https://github.com/TekWizely/pre-commit-golang#go-imports
- https://github.com/TekWizely/pre-commit-golang#go-vet (and "sub"-hooks)

I don't see an alternative to go-unit-tests but... generally all of these repos seem equally dead. Are we getting anything out of this?